.eslintrc.js

```diff
@@ -14,7 +14,7 @@ const rules = {
   'keyword-spacing': [2, { before: true, after: true }],
   'new-cap': [2, { newIsCapExceptions: ['acl.memoryBackend', 'acl'] }],
   'no-caller': 2,
-  'no-console': 2,
+  'no-console': 1,
```

Comment:
I found this blocking compilation to be super annoying since I wanted to test if e.g. my props were being passed from angular with a console.log

maybe there's a cheeky way to override locally?

Reply:
Yep, agreed. There are ways depending on your IDE but what do you think adding this to error on production, and warn on development?

You can use `process.env.NODE_ENV === 'development'` (or `production`|`test`) to detect environment.

Reply:
Made a separate PR for this: https://github.com/Trustroots/trustroots/pull/2456 

I haven't tested it properly yet tho, testing welcome!